docs/development/build-instructions-gn.md

```diff
@@ -65,7 +65,9 @@ origin URLs.
 $ cd src/electron
 $ git remote remove origin
 $ git remote add origin https://github.com/electron/electron
-$ git checkout master
```

Comment:
this can just be updated to `main` - no need to fetch and pull/rebase.

```bash
$ cd src/electron
$ git remote remove origin
$ git remote add origin https://github.com/electron/electron
$ git checkout main
$ git branch --set-upstream-to=origin/main
$ cd -
```

Reply:
If you don't run `git fetch`, then `git checkout main` will definitely fail

Reply:
Strangely enough, I just retried it and found that the problem doesn't exist anymore. I will turn off this PR